Spanner/src/SpannerClient.php

```diff
@@ -193,6 +193,8 @@ class SpannerClient
      *           {@see \Google\Cloud\Spanner\V1\DirectedReadOptions}
      *           If using the `replicaSelection::type` setting, utilize the constants available in
      *           {@see \Google\Cloud\Spanner\V1\DirectedReadOptions\ReplicaSelection\Type} to set a value.
+     *     @type bool $routeToLeader Enable/disable Leader Aware Routing.
```

Comment:
Does the behaviour of some calls change when the `routeToLeader` is `true`/`false`? If so, then I think we should have some system tests.

Reply:
When it is set to true, it will be faster. We may not be able to integration test on this case.

Reply:
Got it.